lightly/api/api_workflow_collaboration.py

```diff
@@ -0,0 +1,76 @@
+from os import access
+from typing import List
+
+from lightly.openapi_generated.swagger_client.models.create_entity_response import CreateEntityResponse
+from lightly.openapi_generated.swagger_client.models.dataset_create_request import DatasetCreateRequest
+
+from lightly.openapi_generated.swagger_client.models.shared_access_config_create_request import SharedAccessConfigCreateRequest
+from lightly.openapi_generated.swagger_client.models.shared_access_config_data import SharedAccessConfigData
+from lightly.openapi_generated.swagger_client.models.shared_access_type import SharedAccessType
+
+from lightly.openapi_generated.swagger_client.models.dataset_data import DatasetData
+from lightly.openapi_generated.swagger_client.rest import ApiException
+from lightly.openapi_generated.swagger_client.models.dataset_type import DatasetType
+
+class _CollaborationMixin:
+
+    def share_dataset(self, dataset_id: str, user_emails: List[str]):
```

Comment:
allow to pass in access_type but default to SharedAccessType.WRITE

Reply:
Same here. I'll create a follow up issue to discuss this.

Reply:
it would just make this directly future proof without needing to touch the pip package.